site/en/blog/policy-update-kwspam-and-circumvention/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Chrome Extensions: Clarifying Our Extension Policies for a More Consistent Web Store"
+description: >
+  Updating our security requirements and further clarifying a few policies to keep the quality of
+  extensions high, and the experience for developers consistent.
+layout: "layouts/blog-post.njk"
+authors:
+  - rebeccasoares
+  - ackermanb
+date: 2021-09-27
+hero: image/SHhb2PDKzXTggPGAYpv8JgR81pX2/9fCG7BsR8e5Md58xiGMV.jpg
+alt: The Roman aqueduct at Segovia, Spain
+tags:
+  - extensions
+
+---
+
+Over the years we've made a number of product and policy improvements to help ensure a quality
+experience for both users and developers. As part of this work, we've updated our best practices,
+and identifiednamed undesirable behaviors that create spammy experiences for users. Today, we're
+furthering this work by clarifying and updating two policies, to keep the quality of extensions
+high, and the experience for developers consistent:
+
+
+## Keyword Spam Update
+
+1.  Keyword Spam is the practice of including irrelevant or excessive keywords in an extensions
+    description in an attempt to manipulate its ranking, resulting in a spammy, negative user
+    experience.
+    
+    We don't allow extensions with misleading, improperly formatted, non-descriptive, irrelevant,
+    excessive, or inappropriate metadata, including but not limited to the extension's description,
+    developer name, title, icon, screenshots, and promotional images.
+    
+    Developers should focus on providing a clear and well-written description that uses keywords
+    appropriately and in context.
```

Comment:
```suggestion
1.  Keyword Spam is the practice of including irrelevant or excessive keywords in an extensions
    description in an attempt to manipulate its ranking, resulting in a spammy, negative user
    experience. We don't allow extensions with misleading, improperly formatted, non-descriptive,
    irrelevant, excessive, or inappropriate metadata, including but not limited to the extension's
    description, developer name, title, icon, screenshots, and promotional images. Developers
    should focus on providing a clear and well-written description that uses keywords appropriately
    and in context.
```

These line breaks still look weird on the preview page. I think it would be best to remove them.

Reply:
I actually wanted these to be paragraphs with _more_ space between them, but I can live with the one giant para.

Reply:
I would also prefer that, but we'd need to work with the site team to get some style changes to support this and I don't think we have the time for that.